cmd/containerd-shim-runc-v2/process/io_test.go

```diff
@@ -67,32 +68,30 @@ func TestNewBinaryIOCleanup(t *testing.T) {
 	}
 }
 
-func descriptorCount(t *testing.T) int {
+func descriptorCount(t *testing.T) (count int) {
```

Comment:
Why is a named return value used?

Reply:
Merely to save on `count := 0` or `var count int` line. Let me know if you don't like it, I can remove it.

Reply:
Removed